README.md

```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
+
+### Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node [here](https://nodejs.org/en/)
```

Comment:
I believe that your intention here is to discuss background knowledge, would a knowledge in only Nodejs be sufficient

Reply:
I have added more information on what the user needs to know to work on the project.